pyproject.toml

```diff
@@ -1,22 +1,25 @@
-[[source]]
-url = "https://pypi.org/simple"
-verify_ssl = true
-name = "pypi"
+[tool.poetry]
+name = "sir-lancebot"
+version = "0.1.0"
+description = "A Discord bot designed as a fun and beginner-friendly learning environment for writing bot features and learning open-source."
+authors = ["Your Name <you@example.com>"]
```

Comment:
If we want to go forth with this PR, this line will have to be changed :D

Reply:
Lol, I chose ignore authors and it does this.

Reply:
Addressed in 1cb0b203bb3af2f9377786aae672593783946314